src/test/java/com/snack/news/fixture/NewsFixture.java

```diff
@@ -1,8 +1,10 @@
 package com.snack.news.fixture;
 
 import com.snack.news.domain.news.News;
+import com.snack.news.dto.AdminNewsDto;
 import com.snack.news.dto.NewsDto;
 import org.junit.Before;
+import sun.tools.jconsole.AboutDialog;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused import - sun.tools.jconsole.AboutDialog.](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4677893)